internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -95,7 +95,7 @@ func (ksd KubernetesServiceDeployer) SetUp(ctxt ServiceContext) (DeployedService
 
 	ctxt.Name = kind.ControlPlaneContainerName
 	ctxt.Hostname = kind.ControlPlaneContainerName
-	ctxt.Agent.Host.NamePrefix = "kind-fleet-agent-"
+	ctxt.Agent.Host.NamePrefix = "kind-control-plane"
```

Comment:
I looked into the kubectl dump for the failed build: https://beats-ci.elastic.co/job/Ingest-manager/job/elastic-package/job/PR-332/1/artifact/build/kubectl-dump.txt

and I see that there is a mix of `kind-control-plane` and `kind-fleet-agent-clusterscope`. I admit I'm bit lost here.